app/frontend/src/aquifers/components/AquiferMap.vue

```diff
@@ -6,6 +6,7 @@
 import L from 'leaflet'
 import 'leaflet-gesture-handling'
 import { tiledMapLayer } from 'esri-leaflet'
+import { uniq } from 'lodash'
```

Comment:
I'm guessing lodash was already installed but I think I would prefer to install just `lodash.uniq` instead and not load the complete library.

Reply:
I see that we are using specific lodash stuff in other places but this isn't the only place that has imported `from 'lodash'`. I feel like we will end up with a lot of package.json pollution by `npm -i lodash.*` libs as needed.

We should discuss and maybe we can find a nicer way of slimming down the lodash dep size using a Webpack/Babel plugin to do the hard work for us.

Reply:
I installed lodash-es which w/ Webpack 4's tree shaking reduced the lodash size in the bundle from 188kb to 44kb. I also aliased moment to moment/src/moment which includes only the EN locale reducing it's size from 193kb to 59kb.